activity/s3/activity.go

```diff
@@ -64,6 +70,8 @@ func (a *Activity) Eval(ctx activity.Context) (done bool, err error) {
 		return false, err
 	}
 
+	logger = ctx.Logger()
+
 	var s3err error
 	switch in.Action {
```

Comment:
We should also allow someone to specify the action in the settings (since most of the time it will probably be static).  But let's leave it in input so they can override the setting with the input one if they desire.